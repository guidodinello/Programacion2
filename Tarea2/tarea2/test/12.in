# Implementar borrarSegmento
borrarSegmento 1 1
insertarSegmentoDespues 1 (3,2.4)
borrarSegmento 1 1
imprimirCadena
inicioCadena
insertarSegmentoDespues 5 (5,3.4) (6,4.1) (7,3.5) (8,9.4) (9,2.6)
borrarSegmento 2 2
imprimirCadena
borrarSegmento 3 4
imprimirCadena
borrarSegmento 1 1
imprimirCadena
inicioCadena
insertarSegmentoDespues 5 (5,3.4) (6,4.1) (7,3.5) (8,9.4) (9,2.6)
imprimirCadena
borrarSegmento 1 6
imprimirCadena
Fin
