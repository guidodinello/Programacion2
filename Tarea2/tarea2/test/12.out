1>#  Implementar borrarSegmento.
2>Segmento borrado.
3>Segmento insertado después de loc.
4>Segmento borrado.
5>
6>loc al inicio.
7>Segmento insertado después de loc.
8>Segmento borrado.
9>(5,3.40)(7,3.50)(8,9.40)(9,2.60)
10>Segmento borrado.
11>(5,3.40)(7,3.50)
12>Segmento borrado.
13>(7,3.50)
14>loc al inicio.
15>Segmento insertado después de loc.
16>(7,3.50)(5,3.40)(6,4.10)(7,3.50)(8,9.40)(9,2.60)
17>Segmento borrado.
18>
19>Fin.
